<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="placeholderText">
    <string>input ip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>port</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditWrite">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_en">
      <property name="text">
       <string>English</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cn">
      <property name="text">
       <string>chinese</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
